src/main/java/hw_3/Task2.java

```diff
@@ -0,0 +1,40 @@
+package hw_3;
+
+import java.util.Arrays;
+
+public class Task2 {
+    public static void main(String[] args) {
+        //ДЗ масиви
+        //Створіть масив із цілих чисел і помістіть відразу 10 значень. Пройдіться ними і змініть значення.
+        int[] array = new int[10];
+        array[0] = 1;
+        array[1] = 2;
+        array[2] = 3;
+        array[3] = 4;
+        array[4] = 5;
+        array[5] = 6;
+        array[6] = 7;
+        array[7] = 8;
+        array[8] = 9;
+        array[9] =10;
+        System.out.println(Arrays.toString(array));
+        //Створіть масив із дробових чисел (5 елементів)
+        double[] array_1 = new double[5];
```

Comment:
Use more specific name for array. array_1 s a bad name

Reply:
"numbers" is ok?

Reply:
yes